integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -157,16 +157,17 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
     assert(result === "hello")
   }
 
-  test("run scala jobs") {
+  test("run echo jobs") {
```

Comment:
Actually it is not necessary to test Scala, because we don't use livy-scala-api, also as I mentioned, involve Scala code in test-lib will potentially introduce issue.